### DISCUSSION TITLE
How to unhide it?

### DISCUSSION THREAD

**Discussion by Arrow75** [2bllw8/anemo#46] [Q&A] [answered]
I used the show in home screen option in the configuration but can't find a way to unhide it

**Comment by 2bllw8** [comment, marked as answer]
Option 1: if you configured a password to unlock the storage, the password input dialog that appears when clicking on the quick tile has a gear icon which leads you to the configuration page.
Option 2: from the settings app of your device, go to the "Apps" section, select the Anemo app and then from there you should find an entry called "Additional settings in the app" (or something similar, depending on the version of Android you're running). That will lead you to the configuration page.

From the configuration page you may re-enable the app icon again
